### DISCUSSION TITLE
Kiosk Mode & RFID - Triggertext in Entry Area # HTML <input>  placeholder

### DISCUSSION THREAD

**Discussion by asaglam76** [kimai/kimai#3973] [General]
Hello, I use the following device for kiosk mode:

- **ALLNET Design LED Tablet 10 Zoll RK3568 (https://tinyurl.com/2p8es8h6)**
From my point of view, this is optimal in terms of price-performance ratio. It also has an integrated NFC scanner which supports MIFARE DESFire EV2 and thus our key tags can also be used for time recording at the same time.

As NFC keyboard I use the Scanner Keyboard from TecIT (Android App). (https://tinyurl.com/3zvuzymd) so that an external reader is not necessary (see manual: https://tinyurl.com/4fyw3kh5).

**However, here is the problem:**
The Scanner Keyboard App supports automatic scanning, but needs a trigger text in the input area (these grayed out texts such as (Email, Password, Token etc.).
![grafik](https://user-images.githubusercontent.com/46278841/231967736-4160476b-a8a5-4cdf-9bcb-f729fb96ebcf.png)

Since there is no trigger text in the input field, the function cannot be used from the app, so the scan has to be started manually.

**My request**:

Is it possible to adapt the plugin so that an individual trigger text or a fixed trigger text can be stored for the input field, so that the app can check for the trigger text and start the autoscan.

![grafik](https://user-images.githubusercontent.com/46278841/231967855-c7ab26e4-6958-4975-958f-7c70f3dc52da.png)

**Comment by kevinpapst** [comment]
Wrong repository, please re-port here: https://github.com/kevinpapst/kimai-plugins

Lets talk over there, sounds super interesting!
